### DISCUSSION TITLE
In Flows -> Web Request: Add an Accept invalid Certificate switch

### DISCUSSION THREAD

**Discussion by biebl** [directus/directus#17757] [Feature Requests]
### Summary

If i want to query an API endpoint, with invalid (selfsigned) Certificate, the Flow fails and i have no opportunity to accept that certificate

### Basic Example

Query a HTTP Endpoint with an invalid certificate => Result:
{
  "message": {
    "message": "self-signed certificate",
    "name": "Error",
    "config": {
      "transitional": {
        "silentJSONParsing": true,
        "forcedJSONParsing": true,
        "clarifyTimeoutError": false
      },
      "adapter": [
        "xhr",
        "http"
      ],
      "transformRequest": [
        null
      ],
      "transformResponse": [
        null
      ],
      "timeout": 0,
      "xsrfCookieName": "XSRF-TOKEN",
      "xsrfHeaderName": "X-XSRF-TOKEN",
      "maxContentLength": -1,
      "maxBodyLength": -1,
      "env": {},
      "headers": {
        "Accept": "application/json, text/plain, */*",
        "Authorization": "Basic <AUTHORIZATION>",
        "Content-Type": "application/JSON",
        "User-Agent": "axios/1.3.3",
        "Accept-Encoding": "gzip, compress, deflate, br"
      },
      "url": "<QUERIEDDOMAIN>",
      "method": "get"
    },
    "code": "DEPTH_ZERO_SELF_SIGNED_CERT",
    "status": null
  }
}

### Motivation

If i have no access to the HTTP Endpoint, i cannot integrate it in my flows

### Detailed Design

If you add an additional ENV Variable for Flows to accept Invalid Certificates, we are done

### Must Have:
- Switch to accept Invalid Certificates

### Drawbacks

None Known

### Alternatives

None Known

### Adoption Strategy

Set default to actual Behaviour

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
